internal/k8s/informers.go

```diff
@@ -31,12 +31,19 @@ import (
 // +kubebuilder:rbac:groups="projectcontour.io",resources=extensionservices,verbs=get;list;watch
 // +kubebuilder:rbac:groups="projectcontour.io",resources=extensionservices/status,verbs=create;get;update
 
+// +kubebuilder:rbac:groups="projectcontour.io",resources=contourconfigurations,verbs=get;list;watch
+// +kubebuilder:rbac:groups="projectcontour.io",resources=contourconfigurations/status,verbs=create;get;update
+// +kubebuilder:rbac:groups="projectcontour.io",resources=contourdeployments,verbs=get;list;watch
```

Comment:
now that i think of it, is access to the `contourdeployment` resource needed in the Contour controller? i *think* only the operator etc. will read/update status on it?

Reply:
I thought we'd retire the operator repo and add a new controller to Contour to manage the Deployment bits, but I do agree that at the moment, it's not being used.

Reply:
Yeah, I think that we should keep this stuff isolated until we start bringing the operator into this repo, so better to leave out for now.

Reply:
Yup makes sense! I took the `ContourDeployment` out for now.